src/bundle/Resources/views/themes/admin/ui/component/dropdown/dropdown.html.twig

```diff
@@ -116,6 +117,18 @@
                     />
                 {% endblock %}
             {%- endembed -%}
+
+            {% if has_select_all_toggler and choices_flat|length >= min_search_items %}
```

Comment:
Not sure if using `min_search_items` variable here is the best idea. It's little confusing, that you want to show toggler based on requirement for search